.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
Include python compiles (`server/__pycache__/**`) and the database (`server/db.sqlite3`).